storage/mysql/schema-mysql.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS SubmittedCerts(
+    ID INTEGER PRIMARY KEY AUTO_INCREMENT,
+    LogID BIGINT NOT NULL,
+    Cert BLOB NOT NULL,
+    SCT BLOB NOT NULL,
+    Timestamp BIGINT NOT NULL,
+    Seen DATETIME
+);
```

Comment:
I think both of these `CREATE TABLE`'s would benefit from explicitly setting the `ENGINE=InnoDB` and the `CHARSET=utf8` like we do in Boulder.